addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -553,6 +553,56 @@ QUnit.test('chat window: open / close', async function (assert) {
     assert.verifySteps(['rpc:channel_fold/open']);
 });
 
+QUnit.test('chat window: focus on ESCAPE', async function (assert) {
+    assert.expect(1);
+
+    this.data['res.partner'].records = [{
+        email: "testpartnert@odoo.com",
+        id: 11,
+        name: "TestPartner",
+    }];
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [
+                {
+                    channel_type: 'channel',
+                    id: 20,
+                    is_minimized: true,
+                    is_pinned: true,
+                    name: "General",
+                    state: 'open',
+                    fold_state: 'open',
+                    uuid: 'channel-20-uuid',
+                },
+                {
+                    channel_type: 'channel',
+                    id: 21,
+                    is_minimized: true,
+                    is_pinned: true,
+                    name: "other chat",
+                    state: 'open',
+                    fold_state: 'open',
+                    uuid: 'channel-21-uuid',
+                }
+            ],
+        },
+    });
+
+    await this.start({
+        hasChatWindow: true
+    });
+
+    await afterNextRender(() => {
+        const ev = new window.KeyboardEvent('keydown', { bubbles: true, key: "Escape" });
+        document.querySelector(`.o_ComposerTextInput_textarea`).dispatchEvent(ev);
+    });
+
+    assert.ok(
+        document.querySelector(`.o_ChatWindow`).classList.contains('o-focused'),
+        "chat window should be focused"
```

Comment:
be more specific in explanations so that the test is really easy to understand by reading them

"next visible chat window should be focused after pressing escape on first chat window"
(this is pretty much the same as the name of the test since there's only one assertion on it for now)